### DISCUSSION TITLE
images upload

### DISCUSSION THREAD

**Discussion by Abdulvoris-Urolov** [outline/outline#5335] [Self Hosting]
![image](https://github.com/outline/outline/assets/80630475/0c107292-e536-4c3b-9d85-28e95cfefcd1)
I'm getting this error and I don't know why

https://docs.getoutline.com/s/125de1cc-9ff6-424b-8415-0d58c809a40f

I applied this and I got the forbidden error gone but I got a 400 error

**Comment by vember31** [comment]
@Abdulvoris-Urolov know this is an old post, but did you ever end up finding a resolution to this issue? I'm also getting a 400 / Bad Request when trying to upload images when using local storage for backend (not using s3)
